aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -115,7 +117,18 @@ module aptos_framework::reconfiguration {
             return
         };
 
-        // Call stake to compute the new validator set and distribute rewards.
+        // Reconfiguration "forces the block" to end, as mentioned above. Therefore, we
```

Comment:
@movekevin @zekun000 So if I understand correctly, on reconfig all subsequent transactions are `Retry` and will be discarded in this block and retried in the next block.

This means that we can actually nicely support cases when validator is removed, but still gets the fees. Also, reconfigurations happen once per block, so from performance point of view aggregator draining here is ok.

Reply:
Yes that'd help with a regular block with a reconfiguration event. This might not work if there's a governance proposal that triggers reconfiguration because there would still be non-zero fees from transactions that happen before the governance execution tx.

Reply:
I remember we had a discussion on this. Can you update this comment here to reflect that discussion?